examples/store.py

```diff
@@ -54,6 +55,30 @@ async def do_upload(account, engine, channel, filename=None, file_hash=None):
         await print_output_hash(message, status)
 
 
+async def do_upload_with_message(account, engine, channel, filename=None, item_hash=None):
```

Comment:
Can you add type annotations ?
```suggestion
async def do_upload_with_message(account: ..., engine: ..., channel: str, filename: Optional[Path]=None, item_hash: Optional[ItemHash]=None):
```